### ISSUE TITLE
get_led_id_for_key_name <class 'TypeError'>: wrong type

### ISSUE THREAD

**Issue by congrongdeyu** [CorsairOfficial/cue-sdk-python#2]
I need soom help.

the code
------------------------------------------------------
from cuesdk import CueSdk

sdk = CueSdk()
sdk.connect()

print("get_devices", sdk.get_devices())

print("get_device_count", sdk.get_device_count())

print("get_device_info", sdk.get_device_info(0))

print("get_last_error", sdk.get_last_error())

print("get_led_id_for_key_name", sdk.get_led_id_for_key_name('B'))
------------------------------------------------------

the output
------------------------------------------------------
get_devices [STRAFE RGB]
get_device_count 1
get_device_info STRAFE RGB
get_last_error CorsairError.Success
Traceback (most recent call last):
  File "C:\Users\dingy\OneDrive\Python\cue-sdk-python-master\mysdkexample\sdk_api_print.py", line 25, in <module>
    print("get_led_id_for_key_name", sdk.get_led_id_for_key_name('B'))
  File "C:\Users\dingy\OneDrive\Python\cue-sdk-python-master\cuesdk\api.py", line 163, in get_led_id_for_key_name
    return native.CorsairGetLedIdForKeyName(key_name)
ctypes.ArgumentError: argument 1: <class 'TypeError'>: wrong type

Process finished with exit code 1
------------------------------------------------------

What is wrong? 
Or, how do I use the function get_led_id_for_key_name?

**Comment by intrueder** [comment]
Hi @congrongdeyu 

In sdk v0.2.5 you can either pass `bytes` literal as an argument or convert string to bytes:

```python
sdk.get_led_id_for_key_name(b'B'))`

key = 'B'
sdk.get_led_id_for_key_name(key.encode()))`
```

This behavior will be changed in future version to allow strings as arguments instead.
Thanks for spotting this!

**Comment by intrueder** [comment]
Fixed in [v0.3.0](https://github.com/CorsairOfficial/cue-sdk-python/releases/tag/v0.3.0)
Now everythyng should work as expected. Your code is valid:
```python
sdk.get_led_id_for_key_name('B')
```

**Comment by congrongdeyu** [comment]
Thank you. @intrueder
